docs/user-guide/solver/05-model.md

```diff
@@ -178,6 +178,21 @@ Note: Almost all variables of the system are defined twice (one value per state)
 | $R\_\lambda \in \mathbb{R}^T_+$                   | stored energy level in reservoir $\lambda$                                                                                                                                         |
 | $\mathfrak{R}\_{\lambda_q} \in \mathbb{R}_+$      | filling level of reservoir layer $q$ at time $T$ (end of the week)                                                                                                                 |
 
+### Short-term storages
+| Notation                                          | Explanation                                                                                                                                                                        |
```

Comment:
For all time varying data, either add subscript `t` in the notation or use the vector form as everywhere else in the notation tables

Reply:
Fixed in 98f69b6d07079974714e7d8282e9abcb9081aae1